src/application/res/import-script/request.gql

```diff
@@ -3,13 +3,13 @@ query {
                     filterFunctions: [
                         {
                             tag: "MdHTimestamp",
-                            value: "2023-05-26 14:11:00",
+                            value: "2023-05-26 14:58:00",
                             operation: GREATER,
                             dataType: TS
                         }
                     ],
                     filterLogicOption: AND,
-                    limit: 1000
+                    limit: 20
```

Comment:
@leondaniel22 @oninach here we say, that we just import 20 data from MdH, right? Do we really want to have this option in the release?

Reply:
@motrell I did n't work on this feature but in my opinion it would be good if we import more data from the MdH.